kernel/contracts/proxy/KernelProxyFactory.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.4.18;
+
+import "./KernelProxy.sol";
+import "../ZepCore.sol";
+import "../KernelRegistry.sol";
+import "../KernelInstance.sol";
+
+contract KernelProxyFactory {
+  event ProxyCreated(address proxy);
+
+  ZepCore private _zepCore;
+
+  function KernelProxyFactory(ZepCore zepCore) public {
+    _zepCore = zepCore;
+  }
+
+  function zepCore() public view returns (ZepCore) {
+    return _zepCore;
+  }
+
+  function createProxy(string distribution, string version, string contractName) public returns (KernelProxy) {
```

Comment:
Should we add a `createProxyAndCall` function, so the contract can be initialized within the same single transaction?